pkg/dataobj/index/calculate.go

```diff
@@ -27,18 +27,14 @@ import (
 type Calculator struct {
 	indexobjBuilder *indexobj.Builder
 	builderMtx      sync.Mutex
-
-	// indexStreamIDLookup is a mapping between the streamID in a logs object & a streamID in the index object.
-	indexStreamIDLookup map[int64]int64
 }
 
 func NewCalculator(indexobjBuilder *indexobj.Builder) *Calculator {
-	return &Calculator{indexobjBuilder: indexobjBuilder, builderMtx: sync.Mutex{}, indexStreamIDLookup: make(map[int64]int64)}
+	return &Calculator{indexobjBuilder: indexobjBuilder, builderMtx: sync.Mutex{}}
```

Comment:
mtx doesn't need to be initialized, zero value is usable

```suggestion
	return &Calculator{indexobjBuilder: indexobjBuilder}
```